core/src/main/java/io/atomix/core/map/AsyncConsistentMap.java

```diff
@@ -213,7 +212,35 @@ default PrimitiveType primitiveType() {
    * @return the previous value (and version) associated with key, or null if there was
    * no mapping for key.
    */
-  CompletableFuture<Versioned<V>> put(K key, V value);
+  default CompletableFuture<Versioned<V>> put(K key, V value) {
+    return put(key, value, Duration.ZERO);
+  }
+
+  /**
+   * Associates the specified value with the specified key in this map (optional operation).
```

Comment:
Is optional operation referring to the TTL?

Reply:
`Duration.ZERO` creates no TTL